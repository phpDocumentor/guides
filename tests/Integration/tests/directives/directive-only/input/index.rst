Directive tests
===============

..  only:: html

    This is only displayed in HTML.

..  only:: tex

    This is only displayed in Latex.

..  only:: html tex

    This is only displayed in both HTML and Latex.
