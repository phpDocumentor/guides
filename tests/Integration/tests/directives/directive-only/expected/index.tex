\documentclass[11pt]{report}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[french]{babel}
\usepackage{cite}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{mathrsfs}
\usepackage{graphicx}
\usepackage{hyperref}
\usepackage{listings}
\begin{document}
\label{}
    \section{Directive tests}

    This is only displayed in Latex.
    This is only displayed in both HTML and Latex.

\end{document}
